simulation/sim/sim_utils.go

```diff
@@ -1,12 +1,16 @@
 package sim
```

Comment:
Why do we need a `sim` intermediary folder?
Looks like files could be directly in simulation